src/cbmc/cbmc_parse_options.cpp

```diff
@@ -430,6 +434,81 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
 
 /*******************************************************************\
 
+Function: cbmc_parse_optionst::unwind_enum_static
+
+  Inputs: goto_functions and options
+
+ Outputs: side effect is adding <clinit> loops to unwindset
+
+ Purpose: unwind static initialization loops of Java enums as far as the enum
+          has elements
+
+\*******************************************************************/
+
+void cbmc_parse_optionst::unwind_enum_static(
+  const goto_functionst &goto_functions,
+  optionst &options)
+{
+  size_t unwind_max=0;
+  forall_goto_functions(f, goto_functions)
+  {
+    auto &p=f->second.body;
+    for(auto it : p.instructions)
```

Comment:
Shouldn't that be `const auto &ins` (i.e, const-ness, reference, and it isn't an iterator hence "it" is not the best choice of a name); on a second thought, const-ness might actually be implied in "auto" here.